Sources/Datadog/Datadog.swift

```diff
@@ -300,11 +300,24 @@ public class Datadog {
 
             Global.crashReporter?.sendCrashReportIfFound()
         }
+
+        deleteV1Folders(in: core)
     }
 
     internal init() {
     }
 
+    private static func deleteV1Folders(in core: DatadogCore) {
+        let deprecated = ["com.datadoghq.logs", "com.datadoghq.traces", "com.datadoghq.rum"].compactMap {
+            try? Directory.cache().subdirectory(path: $0) // ignore errors - deprecated paths likely do not exist
+        }
+
+        core.readWriteQueue.async {
+            // ignore errors
+            deprecated.forEach { try? FileManager.default.removeItem(at: $0.url) }
+        }
+    }
```

Comment:
question/ Just making sure - would this mean that V2 core will be aware of V1 feature folders, right? I think it's reasonable and trying to delete each folder from corresponding V2 feature would be too much hassle 👍.

Reply:
That's right! `Datadog` class is already aware of the v1 feature types, so I think it's fine. I'm not a big fan of hardcoded values out of nowhere like this, but it's a migration step that will ultimately be removed  🤷‍♂️ 

One thing to note is that it now deletes the folders, previously it was only removing the content of the folders.